.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
```

Comment:
A good place to look for configs would be https://github.com/dustinspecker/awesome-eslint#configs.